layers/erspan2.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2018 Google, Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"encoding/binary"
+	"net"
+	"time"
+
+	"github.com/google/gopacket"
+)
+
+// Header contains all of the fields found in an ERSPAN2 header
+// https://tools.ietf.org/html/draft-foschiano-erspan-03
+type Header struct {
+	IsTruncated                         bool
+	Version, CoS, TrunkEncap            uint8
+	VLANIdentifier, SessionID, Reserved uint16
+	Index                               uint32
+}
+
+const (
```

Comment:
Same here, if the const are related only to ERSPANII It must be the prefix of the const names